src/common/res/features/shared/main.js

```diff
@@ -105,6 +105,30 @@ ynabToolKit.shared = new function() {
         }
     },
 
+    // TODO Maybe add universal function.
+    // Usage: declension(daysNumber, {nom: 'день', gen: 'дня', plu: 'дней'});
+    this.declension = function (locale, num, cases) {
```

Comment:
Knowing about languages I know what declension is, but it'd be great if we could put a comment above that explains why this exists and what it's for. Arabic is a good example of a language with a shocking amount of odd declension that would be good to reference.

Monolingual English speakers have a tendency to assume that all languages much just be singular or plural like English is. :-)